gridstatus/eia.py

```diff
@@ -734,20 +753,69 @@ def _handle_fuel_type_data(df):
 
     df["MW"] = df["MW"].astype(float)
 
+    # The raw data will sometimes have case-insensitive duplicates
+    # (e.g. "Pumped Storage","Pumped storage"). We can handle that through the pivot
+    # table by summing the duplicates.
+    df["type-name"] = df["type-name"].str.lower()
+
+    # These columns are grouped together by EIA as confirmed by inspection of the EIA
+    # fuel mix graphs
+    df["type-name"] = df["type-name"].replace(
+        {
+            "battery": "battery storage",
+            "solar battery": "solar with integrated battery storage",
+            "unknown energy": "unknown energy storage",
```

Comment:
These combinations are what EIA does on their charts. It's possible (likely?) EIA will change these again or start publishing additional fuel types.

https://www.eia.gov/electricity/gridmonitor/expanded-view/electric_overview/US48/US48/GenerationByEnergySource-4/edit

Reply:
i think you can leave this comment in the code too